components/post_view/post_time.jsx

```diff
@@ -7,7 +7,8 @@ import {Link} from 'react-router';
 
 import TeamStore from 'stores/team_store.jsx';
 
-import {isMobile, getWindowDimensions} from 'utils/utils.jsx';
+import {isMobile} from 'utils/user_agent.jsx';
+import {getWindowDimensions} from 'utils/utils.jsx';
```

Comment:
getWindowDimensions is used for determining mobile view on resize. If we are going to use user agent i think its safe to remove `getWindowDimensions ` now.

Reply:
Yeah. I am confused as to what getWindowDimensions is even used for in this file? It sets the state, but I don't see the state being used at all...

Reply:
Although that isn't your change, I wonder if you should remove it since you are in the file already?

Reply:
@stephenkiers it is just for [re-rendering](https://github.com/mattermost/mattermost-webapp/pull/435#discussion_r156167921) component on resize

Reply:
So, it is used to trigger a setState, and hence a re-render, when the window size changes? ok...

Reply:
Yes. From ticket looks like this behaviour is not intended now so, was suggesting to remove